### PR TITLE
Implementation of PAFPN & FPG

### PR DESCRIPTION
**Pull request by zehuichen123**
The results have been reported in config/xxx/README.md. We also provided with an advanced version of PAFPN, by stacking top down and bottom up pathways multiple times, which we found it useful on waymo open dataset.

### CODE REVIEW

**Comment by RogerChern** [comment]
There is a TDBU FPN in NASFPN and I think that should be moved here too.

**Comment by zehuichen123** [comment]
> There is a TDBU FPN in NASFPN and I think that should be moved here too.

This TDBU seems a special case of PAFPN when num_stage = 1, so maybe we can just remove it?

**Comment by RogerChern** [comment]
It is a PAFPN with 3 stages and 384 channels. You can append its config and
model to the table of FPG.

**Comment by zehuichen123** [comment]
Alright. I will rerun this TDBU with 3 stages and 384 channels on faster r-cnn to align its settings with FPG and PAFPN and merge it into the table of FPG.

**Comment by zehuichen123** [comment]
@RogerChern Hi, I've added this TDBU neck to FPG and renamed it as PAFPNNeckP3P7 along with its performance on Faster R-CNN.

File: models/FPG/builder.py [hunk 1]
```diff
@@ -0,0 +1,449 @@
+import mxnet as mx
+import mxnext as X
+from symbol.builder import Neck
+from models.NASFPN.builder import NASFPNNeck
+
+def merge_sum(sum_list, name):
```
Comment (RogerChern) [inline models/FPG/builder.py:6]: I've seen this two helper functions so many times across the codebase. How about move them in mxnext?

**Comment by RogerChern** [comment]
Except the repeating codes issue, this PR looks good to me.

**Comment by RogerChern** [comment]
Also don't forget to update the main readme

**Comment by zehuichen123** [comment]
I've moved `merge_sum` and `reluconvbn` to mxnext and gave one PR to mxnext repo.

**Comment by RogerChern** [comment]
lol, when doing refactorization, you shoud also remove the merge_sum and reluconvbn elsewhere like in NASFPN. This could be done by a search in the repo. I will merge the PR in mxnext first.

**Comment by RogerChern** [comment]
So throught this trial and error. I think the following procedure could serve as a brief checklist before merging any PR.

1. provide the module codes
2. provide the module readme, performance, checkpoints and logs.
3. inspect the introduced helper funcitons to see if it is necessary to ship with the module. Search for any duplications in the repo.
4. modify the main readme

**Comment by zehuichen123** [comment]
> lol, when doing refactorization, you shoud also remove the merge_sum and reluconvbn elsewhere like in NASFPN. This could be done by a search in the repo. I will merge the PR in mxnext first.

Yes, I've changed those merge_sum & reluconvbn under NASFPN folder.

**Comment by RogerChern** [comment]
Cool！

**Pull request merged**